packages/datagateway-dataview/src/views/table/investigationTable.component.test.tsx

```diff
@@ -171,7 +171,7 @@ describe('Investigation table component', () => {
     jest.clearAllMocks();
   });
 
-  it('renders correctly', async () => {
+  it.skip('renders correctly', async () => {
```

Comment:
Do we keep these skips in?

Reply:
oops no - they were left over from my debugging